mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+
+from mapie._machine_precision import EPSILON
+from mapie._typing import ArrayLike, NDArray
+
+from mapie.conformity_scores import ConformityScore
+
+
+class AbsoluteConformityScore(ConformityScore):
+    """
+    Absolute conformity score.
+
+    The signed conformity score = y - y_pred.
```

Comment:
Not sure if you want to do this everytime, but I think this is cleaner: `y - y_pred`. But I see you use this multiple times in your docstring.